src/touchcontroller.cs

```diff
@@ -0,0 +1,846 @@
+	[Native]
+	public enum TCControlContentsButtonShape : long {
+		Circle,
+		Rect,
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Native]
+	public enum TCControlContentsDpadDirection : long {
+		Up,
+		Down,
+		Left,
+		Right,
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Native]
+	public enum TCControlContentsDpadElementStyle : long {
+		Circle,
+		Pentagon,
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Native]
+	public enum TCControlLabelRole : long {
+		Button,
+		DirectionPad,
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Native]
+	public enum TCControlLayoutAnchor : long {
+		TopLeft,
+		TopCenter,
+		TopRight,
+		CenterLeft,
+		Center,
+		CenterRight,
+		BottomLeft,
+		BottomCenter,
+		BottomRight,
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Native]
+	public enum TCControlLayoutAnchorCoordinateSystem : long {
+		Relative,
+		Absolute,
+	}
+
+	interface ITCControl { }
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Protocol (BackwardsCompatibleCodeGeneration = false)]
+	interface TCControl : TCControlLayout {
+
+		[Abstract]
+		[Export ("label", ArgumentSemantic.Strong)]
+		TCControlLabel Label { get; }
+
+		[Abstract]
+		[Export ("colliderShape", ArgumentSemantic.Assign)]
+		TCColliderShape ColliderShape { get; }
+
+		[Abstract]
+		[Export ("pressed")]
+		bool Pressed { [Bind ("isPressed")] get; }
+
+		[Abstract]
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Abstract]
+		[Export ("handleTouchBeganAtPoint:")]
+		void HandleTouchBeganAtPoint (CGPoint point);
+
+		[Abstract]
+		[Export ("handleTouchMovedAtPoint:")]
+		void HandleTouchMovedAtPoint (CGPoint point);
+
+		[Abstract]
+		[Export ("handleTouchEndedAtPoint:")]
+		void HandleTouchEndedAtPoint (CGPoint point);
+
+		[Export ("highlightDuration")]
+		double HighlightDuration { get; set; }
+	}
+
+	interface ITCControlLayout { }
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[Protocol (BackwardsCompatibleCodeGeneration = false)]
+	interface TCControlLayout {
+
+		[Abstract]
+		[Export ("anchor", ArgumentSemantic.Assign)]
+		TCControlLayoutAnchor Anchor { get; set; }
+
+		[Abstract]
+		[Export ("anchorCoordinateSystem", ArgumentSemantic.Assign)]
+		TCControlLayoutAnchorCoordinateSystem AnchorCoordinateSystem { get; set; }
+
+		[Abstract]
+		[Export ("offset", ArgumentSemantic.Assign)]
+		CGPoint Offset { get; set; }
+
+		[Abstract]
+		[Export ("zIndex")]
+		nint ZIndex { get; set; }
+
+		[Abstract]
+		[Export ("size", ArgumentSemantic.Assign)]
+		CGSize Size { get; set; }
+
+		[Abstract]
+		[Export ("position", ArgumentSemantic.Assign)]
+		CGPoint Position { get; }
+	}
+
+	[NoMacCatalyst, iOS (26, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface TCButton : TCControl, TCControlLayout {
+
+		[NullAllowed, Export ("contents", ArgumentSemantic.Strong)]
+		TCControlContents Contents { get; set; }
+
+		[Export ("highlightDuration")]
+		new double HighlightDuration { get; set; }
+
+		[Export ("colliderShape", ArgumentSemantic.Assign)]
+		new TCColliderShape ColliderShape { get; }
```

Comment:
IIRC you shouldn't have to re-define members that are otherwise inlined from protocols (unless the signature or availability attributes are different).